### DISCUSSION TITLE
Livros Javascript

### DISCUSSION THREAD

**Discussion by vitocaio** [frontendbr/forum#229] [Discussão]
Ola Pessoal do Fontendbr, vim até aqui hoje para saber se alguém gostaria de compartilhar alguns livros comigo, eu gostaria de ler PROFESSIONAL JAVASCRIPT® FOR WEB DEVELOPERS ou JAVASCRIPT O GUIA DEFINITIVO, porém os preços estão bem altos, se alguém tiver em PDF na versão traduzida em Português 
ajudaria muito, também aceito sugestões de outros livros, mas meu foco é um desses dois mesmo, obrigado!

**Comment by diegoemerique** [comment]
Eu tenho esse Professional JavaScript for Web Developers 3rd Edition. Ele é muito bom, não sei se é melhor do que o guia definitivo, mas certamente é bem completo e fala de tudo sobre javascript.

Sobre ES6, não cheguei a terminar de ler mas n vi falando sobre, pq o livro é de 2012 então é tudo no ES5 então n deve ter muito sobre.

E se for comprar, compre a ultima versão que é a terceira. ;)

**Comment by leonardoelias** [comment]
Eai @vitocaio o JAVASCRIPT O GUIA DEFINITIVO eu tenho ele aqui 👍  
Se o @LFeh fazer um desafio fizer um desafio para você e cumprir eu te mando ele haha ( você pagando o frete claro )

**Comment by vitocaio** [comment]
Seria em PT-BR?
se for eu pago o frete tranquilo rs.

@diegoemerique você não tem ele em PDF?
eu queria o livro emprestado mesmo rs, estou com poca grana, a ideia é que alguem que possua o livro possa emprestar, claro que vou assumir o frete e a responsabilidade de cuidadar do livro

**Comment by leonardoelias** [comment]
@vitocaio sim em PT-BR

**Comment by vitocaio** [comment]
Legal!
bom se puder disponibilizar, ficaria muito agradecido, vai ser de muita importância para o meu desenvolvimento com a linguagem.

**Comment by fabricionaweb** [comment]
@LFeh tive uma idéia aqui

podia criar um repo (como o fórum e vagas) para doações de livros 😬

**Comment by vitocaio** [comment]
@o@ 
Malandro! que ideia fantástica! eu tenho bons livros para doar, bom não de programação em si, mas tenho rs.

**Comment by ingridrauany** [comment]
Esse [projeto](https://github.com/ericdouglas/traduz-ai) tem uns artigos traduzidos sobre JS e estão traduzindo a série "You don't know JS" (que eu estou gostando bastante).

**Comment by paulinhoerry** [comment]
@vitocaio, recomendo fortemente o "You don't know JS", o livro já está traduzido para PT-BR e está disponível gratuitamente nesse [repositório](https://github.com/cezaraugusto/You-Dont-Know-JS)

**Comment by vitocaio** [comment]
Opa parece exatamente o que procuro rs.

**Comment by cmpereirasi** [comment]
#### Muito show galera!

Também tem o [Eloquente JS](https://github.com/braziljs/eloquente-javascript) completa, já traduzido para **PT-BR**

**Comment by vitocaio** [comment]
Alguém ai conhece algum livro bom para NodeJS também?

**Comment by felipefialho** [comment]
@fabricionaweb Ótima ideia!

@frontendbr/admins O que acham?

**Comment by diogomoretti** [comment]
Acho bem bacana a ideia. Pode ser doação + livros free da web =]

**Comment by vitocaio** [comment]
Opa que bom que esse tópico incentivou essa maravilhosa ideia, já estava cansado de tentar adivinhar o nome de bons livros kkk

**Comment by felipefialho** [comment]
Done pessoal. Compartilhem e façam bombar -> https://github.com/frontendbr/doe-um-livro

Valeu pela sugestão @fabricionaweb ;)

**Comment by dleitee** [comment]
@lfvirtuoso na maioria dos repositórios existe um passo-a-passo de como contribuir, geralmente via pull resquest.

**Comment by herberthenrique** [comment]
Eu gosto bastante do[ Segredos do Ninja Javascript](http://www.buscape.com.br/segredos-do-ninja-javascript-john-resig-bear-bibeault-8575223283)

É bom para quem está começando e aprendendo (:

**Reply by LaraGb** [reply]
Tem o link?

**Comment by raisiqueira** [comment]
React - Guia do Iniciate (PT-BR)
Aprenda React de forma fácil e progressiva, compreendendo cada detalhe da biblioteca a cada exemplo apresentado.
Gostei desse livro, é uma abordagem bem "Hello World" para quem quer entrar no mundo do React. 
https://leanpub.com/react-guia-do-iniciate

**Comment by vitocaio** [comment]
@raisiqueira muito bom! tem muito material bacana, vlw pessoal

PS: se tiverem mais livros para divulgar, divulguem ou doem pelo   https://github.com/frontendbr/doe-um-livro

Muito obrigado pela contribuição de todos!

**Comment by felipefialho** [comment]
@vitocaio Lembrando que só vale eBooks de livros gratuitos ou doação. Divulgação não :)

**Comment by vitocaio** [comment]
Ops!
Retratando, doações ou compartilhamento de livros gratuitos!

@LFeh

**Comment by leoigel** [comment]
sera que podem me ajudar?? to procurando o livros segredos do ninja em javascript, Padrão Javascript e todas as series do livro you don't know js ,porem tem que ser em portugues!!! e em pdf pois nao tenho como comprar no brasil(posso pagar tambem)

**Reply by LaraGb** [reply]
Você conseguiu? Estou procurando tbm

**Comment by SandroSena** [comment]
[Aqui](https://github.com/cezaraugusto/You-Dont-Know-JS) tem a serie do YDKJS em português completa se não me engano.
